sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -21,7 +21,13 @@ This product is fully supported and enabled by default as of {product-title} 4.1
 [id="sandboxed-containers-1-3-bug-fixes"]
 == Bug fixes
 
+* Previously, when creating the `KataConfig` CR and observing the pod status under the `openshift-sandboxed-containers-operator` namespace, a huge number of restarts for monitor pods was shown. The monitor pods use a specific SELinux policy that was installed as part of the `sandboxed-containers` extension installation. The monitor pod was created immediately, however the SELinux policy was not yet available, which resulted in a pod creation error, followed by a pod restart.
```

Comment:
Is 'CR'assumed to be known by the audience? I couldn't find it in our guides, so just want to check.

Reply:
@tmalove CR is [custom resource](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#custom-resource) and it's widely used

Reply:
You could break up this long sentence to help with localization.

The monitor pod was created immediately. However, the SELinux policy was not yet available . . .